pkg/virtualKubelet/forge/forge.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019-2021 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package forge
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ReflectionFieldManager -> The name associated with the fields modified by virtual kubelet reflection.
+const ReflectionFieldManager = "reflection.liqo.io"
+
+var (
+	// LocalClusterID -> the cluster ID associated with the local cluster.
```

Comment:
Cosmetics 💄  Do we use this style for comments, with the `->`,  anywhere else?

Reply:
Sometimes yes, because the linter complains if using colons